open_fdd/validation/paired_fdd_parity.py

```
@@ -0,0 +1,106 @@
+
+
+def masks_equal(
+    left: pa.Array | pa.ChunkedArray,
+    right: pa.Array | pa.ChunkedArray,
+    *,
+    float_tol: float = 1e-9,
+) -> bool:
+    if len(left) != len(right):
+        return False
+    if pa.types.is_floating(left.type) or pa.types.is_floating(right.type):
+        l = pc.cast(left, pa.float64())
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)